reports/emulator_base64bz2.go

```diff
@@ -0,0 +1,38 @@
+package reports
+
+import (
+	"log"
+)
+
+func BuildBase64bz2Report() (Base64bz2Report, error) {
+
+	machineInfo, err := EmulateMachineInfo()
+	if err != nil {
```

Comment:
With all these errors, don't log them here, just return them. 

I also suggest introducing the `https://github.com/pkg/errors` package as a dependency. It helps annotate your errors and provide some additional context. 
(with `github.com/pkg/errors`):

```
machineInfo, err := EmulateMachineInfo()
if err != nil {
    return Base64bz2Report{}, errors.Wrap(err, "machine info")
}
```

The caller will also check the error and can decide what to do.
Applies to all the errors below.